examples/nimble_blecent/peer.c

```diff
@@ -0,0 +1,926 @@
+/****************************************************************************
+ * apps/examples/nimble_blecent/peer.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <assert.h>
+#include <string.h>
+#include "host/ble_hs.h"
+#include "blecent.h"
+
+static void *peer_svc_mem;
+static struct os_mempool peer_svc_pool;
+
+static void *peer_chr_mem;
+static struct os_mempool peer_chr_pool;
+
+static void *peer_dsc_mem;
+static struct os_mempool peer_dsc_pool;
+
+static void *peer_mem;
+static struct os_mempool peer_pool;
+static SLIST_HEAD(, peer) peers;
+
+static struct peer_svc *
+peer_svc_find_range(FAR struct peer *peer, uint16_t attr_handle);
+static struct peer_svc *
+peer_svc_find(FAR struct peer *peer, uint16_t svc_start_handle,
+              struct peer_svc **out_prev);
+int peer_svc_is_empty(const struct peer_svc *svc);
+
+uint16_t
+chr_end_handle(FAR const struct peer_svc *svc, FAR const struct peer_chr *chr);
```

Comment:
Yeah, line is too long.. please take a look at https://nuttx.apache.org/docs/latest/contributing/coding_style.html :-)